packages/Graph/Schemas/Microsoft.Graph.Calendar.CreateEvent.schema

```diff
@@ -26,25 +26,105 @@
       ]
     },
     "token": {
-      "$ref": "schema:#/definitions/valueExpression",
+      "$ref": "schema:#/definitions/stringExpression",
       "title": "Token",
       "description": "Microsoft Graph API authentication token.",
       "examples": [
         "=turn.token.token"
       ]
     },
-    "eventToCreateProperty": {
-      "$ref": "schema:#/definitions/valueExpression",
+    "eventToCreate": {
+      "$ref": "schema:#/definitions/objectExpression",
```

Comment:
>"$ref": "schema:#/definitions/objectExpression", [](http://example.com/codeflow?start=6&length=48)

I think the more correct description is to drop this.

Reply:
Resolved